_docs/setup/install-kubernetes.md

```diff
@@ -18,10 +18,10 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
 The following instructions assume you have access to a Kubernetes **1.7.4 or newer** cluster
 with [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled.
 
-  > Note: Istio 0.2.x is not backward compatible with 0.1.x. Uninstall any 0.1.x
-  > version of Istio before installing the newer version. Any applications
-  > using Istio sidecar from 0.1.x version must upgrade to using the
-  > sidecar supplied by 0.2.x release.
+  > Note: If you installed Istio 0.1.x please
+  > [uninstall](https://istio.io/v-0.1/docs/tasks/installing-istio.html#uninstalling)
+  > it completely before installing the newer version (including the Istio sidecar
```

Comment:
phrase in parentheses is vague--does this mean 'you need to uninstall the sidecar too', or 'you need to uninstall Istio before installing the newer version and/or the sidecar'